lib/spring/generate/springInitializrGenerator.ts

```diff
@@ -34,32 +34,36 @@ export function addSpringInitializrGenerator(sdm: SoftwareDeliveryMachine) {
         intent: "spring initializr",
         description: "Create a new Spring Boot project using Spring Initializr",
         parameters: SpringInitializrProjectCreationParameterDefinitions,
-        startingPoint: springInitializrProject,
+        startingPoint: params => springInitializrProject(sdm, params),
         transform: [
             SetAtomistTeamInApplicationYml,
             TransformSeedToCustomProject,
         ],
     });
 }
 
-function springInitializrProject(params: SpringInitializrProjectCreationParameters): Promise<Project> {
+function springInitializrProject(sdm: SoftwareDeliveryMachine, params: SpringInitializrProjectCreationParameters): Promise<Project> {
     const url = "https://start.spring.io/starter.zip";
     const tmpDir = tmp.dirSync({unsafeCleanup: true});
     const cwd = tmpDir.name;
     const lastSlash = url.lastIndexOf("/");
     const filename = url.substring(lastSlash + 1);
     const zipFile = path.join(cwd, filename);
     const urlParameters = getUrlParameters(params);
-    return downloadFileAs(url, zipFile, urlParameters)
+    const headers = {
+        "User-Agent": "atomist/" + sdm.configuration.name + "-" + sdm.configuration.name,
```

Comment:
Why the name twice? Was that intended to be `sdm.configuration.name + "-" + sdm.configuration.version`?

Reply:
Already saw it, already fixed